libobs/obs-encoder.c

```diff
@@ -678,7 +678,7 @@ void obs_encoder_shutdown(obs_encoder_t *encoder)
 	if (encoder->context.data) {
 		encoder->info.destroy(encoder->context.data);
 		encoder->context.data = NULL;
-		encoder->paired_encoder = NULL;
+		da_clear(encoder->paired_encoders);
```

Comment:
Is that enough? Wouldn't it be more correct to call `darray_free(encoder->paired_encoders)` (it clears implicitly) to avoid a memory leak?